mirage/config.js

```diff
@@ -64,8 +64,15 @@ export default function () {
     return schema.organizations.all();
   });
 
-  this.get('/user', function (schema) {
-    return this.serialize(schema.users.first(), 'v3');
+  this.get('/user', function (schema, request) {
+    const { authorization } = request.requestHeaders;
+    const firstUser = schema.users.first();
+
+    if (authorization !== `token ${firstUser.token}`) {
+      return new Response(403, {}, {});
+    }
+
+    return this.serialize(firstUser, 'v3');
```

Comment:
When I turned off the jquery integration, it broke the automatic signout test. In the test, the token expired flash message was no longer showing up, because the `fetchCurrentUser` task in the `auth` service was not getting an error from `currentUser.reload`. I tried on production changing my token in localStorage and reloading the page while logged in, and noticed that `/user` returns with a `403`, which seems to allow the auto signout to work.

So I made this update to mirage config, which returns 403 if token doesn't match. I'm not really sure why this test works at all when jquery integration is enabled?

Reply:
Nice, @sorta thanks for catching this 👍